MusicTaste/MusicTaste/View Controllers/MatchmakingViewController.m

```diff
@@ -44,8 +46,9 @@ - (void)viewDidLoad {
 - (void)updateView:(BOOL)animated{
     //Function to update matches once the notifcation is returned
     self.author  = [SpotifyAPIManager shared].author;
-    [self getMatchesDictionary];
-    [[MatchingAlgorithm shared] lookForMatches];
+    if ([self.author valueForKey:@"matches"] != nil){
```

Comment:
Checks if matches aren't nil before looking for matches again

Reply:
Nice!